### ISSUE TITLE
Windows cannot find exe

### ISSUE THREAD

**Issue by ehernan3** [tripflex/MOD-t#5]
A friend has given me their Mod-t printer. I then learned that NewMatter has closed, so I found your git repo, and downloaded MOD-t-FINAL-DISABLES-WIFI.exe from this repo. I'm trying to run this on my Windows 10 laptop.

When I double click the file, a window pops up "This app can't run on your PC. To find a version for your PC, check with the software publisher".

When I Run as administrator, I get the following message, which contradicts my file system:
![Capture](https://user-images.githubusercontent.com/39863727/101910756-65d7c880-3b8d-11eb-9a94-1722fac65acc.JPG)

Is this software not compatible with Windows 10?

**Comment by tripflex** [comment]
Download looks corrupted, it should be almost 50mb in size, try downloading with a different browser

**Comment by ehernan3** [comment]
@tripflex , this issue is resolved. What I had done earlier was RMB Save As on the exe file so I could choose the destination directory. That apparently doesn't work in Git, so(using the same browser as before, Chrome) I clicked on the file to expose the Download button, which works flawlessly.

You were totally correct that the original Save As download was corrupted. Click on the file first shows that it is indeed 43Mb, yet Save As corrupts that into 100k. Shame on me for not noticing that.

Hmm, I didn't know that about Git. I use it at work, but at work, I never 'download' stuff. I'm always following the normal Gitflow(cloning, committing, pushing, merging, etc). This is the first time I've tried to use Git outside the normal flow.
